README.md

```diff
@@ -695,8 +770,23 @@ Private properties should be declared in class extensions (anonymous categories)
 @end
 ```
 
+## Private headers
+
+Use private headers to define class extension and any internal methods and properties which can be used for [unit tests](#unit-tests). Also they have to contain all necessary inclusions.
```

Comment:
What is "All necessary inclusions"? Should "_Private.h" header contain imports that needed in .m file only?